Editor/NRepl.cs

```diff
@@ -312,6 +352,7 @@ static void HandleMessage (BDictionary message, TcpClient client)
 		{
 			var opValue = message["op"];
 			var opString = opValue as BString;
+			var autoCompletionSupportEnabled = (bool)((IPersistentMap)configVar.invoke()).valAt(Keyword.intern("nrepl-auto-completion"));
```

Comment:
I would use `clojure.lang.RT.booleanCast` in place of the `bool` cast here. It implements Clojure truthiness correctly. If a user puts a non-convertible value into their config, like `:nrepl-auto-completion "yes"`, this would throw an exception. An edge case for sure, but one we can account for.